components/piper.yaml

```diff
@@ -55,10 +50,7 @@ services:
       - monitoring
       - database
       - services
-    volumes:
-      - chrome-share:/tmp/share
-      - packages:/app/packages
-    command: ["worker"]
+    entrypoint: ["./worker.sh"]
```

Comment:
I have an idea. We can use makefile and add all entrypoints to it. In Dockerfile entry point = "make", and command in compose simply "worker". It will be unified. I can assist to you here